### PR TITLE
Added CustomTextField and exposed name and label props for dropdowns.

### PR DESCRIPTION
**Pull request by thaigillespie**

### CODE REVIEW

File: src/components/CustomTextField/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TextField } from '@material-ui/core';
+
+const CustomTextField = props => {
```
Comment (godwinpang) [inline src/components/CustomTextField/index.js:6]: hm let's take some time to think about naming - don't think CustomTextField looks the nicest in code but then TextField might be confusing

**Comment by godwinpang** [comment]
feel free to just merge it in but take a look at the requested changes

File: src/components/CustomTextField/index.js (hunk 1 above)
Comment (thaigillespie) [reply src/components/CustomTextField/index.js:6]: I honestly don't know what's a better name... MuiTextField maybe? Not sure : (

Comment (godwinpang) [reply src/components/CustomTextField/index.js:6]: Ah naming - the hardest part of software engineering

Comment (thaigillespie) [reply src/components/CustomTextField/index.js:6]: Okay how about InputField?

Comment (godwinpang) [reply src/components/CustomTextField/index.js]: yeah man idk - I think it should be textfield but I guarantee it'll be confusing for people

Comment (godwinpang) [reply src/components/CustomTextField/index.js]: idt MuiTextField is good cuz this is *our* custom TextField

Comment (thaigillespie) [reply src/components/CustomTextField/index.js]: Yepppp 😩😩

Comment (godwinpang) [reply src/components/CustomTextField/index.js]: Let's finalize something tonight and stick with it for a bit

**Comment by thaigillespie** [comment]
I just decided to stick with InputField for now, check my most recent commit

**Comment by godwinpang** [comment]
Oh I like InputField a lot

**Comment by godwinpang** [comment]
merge it in!

**Pull request merged**